src/main/java/org/launchcode/techjobs/persistent/controllers/EmployerController.java

```diff
@@ -1,18 +1,35 @@
 package org.launchcode.techjobs.persistent.controllers;
 
 import org.launchcode.techjobs.persistent.models.Employer;
+import org.launchcode.techjobs.persistent.models.Job;
+import org.launchcode.techjobs.persistent.models.data.EmployerRepository;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Controller;
 import org.springframework.ui.Model;
 import org.springframework.validation.Errors;
 import org.springframework.web.bind.annotation.*;
 
 import javax.validation.Valid;
+import java.util.List;
 import java.util.Optional;
 
+
 @Controller
-@RequestMapping("employers")
+@RequestMapping("employer")
```

Comment:
```suggestion
@RequestMapping("employers")
```
This should be changed back to `employers` because the links in the template for Job's `add.html` expect `employers/add` in the URL path.